<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatabaseWidget</class>
 <widget class="QWidget" name="DatabaseWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="m_pResizeList">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>410</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>resize list</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="m_pListCount">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>410</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>99999</number>
   </property>
  </widget>
  <widget class="QTreeWidget" name="m_pList">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>201</width>
     <height>401</height>
    </rect>
   </property>
   <property name="indentation">
    <number>0</number>
   </property>
   <property name="rootIsDecorated">
    <bool>false</bool>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <property name="headerHidden">
    <bool>false</bool>
   </property>
   <property name="expandsOnDoubleClick">
    <bool>false</bool>
   </property>
   <property name="columnCount">
    <number>2</number>
   </property>
   <attribute name="headerVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="headerCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="headerDefaultSectionSize">
    <number>50</number>
   </attribute>
   <attribute name="headerHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="headerShowSortIndicator" stdset="0">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>name:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="m_pID">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>0</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="maximum">
    <number>999999999</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_42">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>0</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ID:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_pName">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
